dsl/camel-jbang/camel-jbang-plugin-k/pom.xml

```diff
@@ -46,6 +46,13 @@
           <artifactId>camel-jbang-core</artifactId>
         </dependency>
 
+        <!-- TODO: clarify why jbang plugin is not part of camel-bom -->
```

Comment:
Add a JIRA so we can add it to camel-bam

Reply:
👍 https://issues.apache.org/jira/browse/CAMEL-20955